### ISSUE TITLE
Set License, Confirm License Plans

### ISSUE THREAD

**Issue by TheOtherZech** [TheOtherZech/vada#5]
**TL;DR:** My ideal licensing setup for Vada is a dual license scheme offering both MPL 2.0 and closed-fork-compatible versions *of the evaluator*, functioning as a sort of "share code or fund the project" setup. Is that practical? Probably not.

To be clear, the license discussion here is for the implementation, not the specification. The formal specification has its own silo for licensing concerns, with the whole "source of truth" and dialect-identification thing to deal with. 

My main concern with any sort of dual licensing setup is that it will lead to ambiguity, limiting usage. I want Vada to be used in both closed source and open source projects. I want to people to hack on and extend the language itself. 

But it's easy to imagine situations where users look at the licensing setup and **erroneously** believe they need to share:

* **All** of their source code, rather than just the source of the modified evaluator they ship
*  The source code for evaluator modifications they *don't* ship
*  The source Vada archives produced by a modified evaluator, independent of whether they ship that evaluator
*  Auto-generated ingest code for Vada archives, independent of whether that code is generated by a modified tool

I want to prevent the above misconceptions wherever possible, but doing that depends on how familiar folks are with MPL 2.0 licensing. This might be a situation where a more common license, even if it's more permissive, would ultimately be better for Vada as a whole.
